### PR TITLE
Link dynamically with the Vulkan loader

### PR DESCRIPTION
**Pull request by scribam**
It is not recommended to link with VKstatic.1.lib from the Vulkan SDK (source: https://vulkan.lunarg.com/issue/view/5a70a80a6631511d3732aa03) so let's load dynamically the system vulkan-1.dll instead.

It could be required to download and install the Vulkan Runtime (from https://vulkan.lunarg.com/sdk/home) but generally, it is shipped when you install your graphic card driver.

It helps with the issue #4625 (at least for @Psycho-A)

Also, CMake on Windows does the same (linking with vulkan-1.lib, https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindVulkan.cmake#L40)

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ environment:
   GLSLANG: https://drive.google.com/uc?export=download&id=1A2eOMmCO714i0U7J0qI4aEMKnuWl8l_R
   COMPATDB: https://rpcs3.net/compatibility?api=v1&export
   VULKAN_SDK: "C:\\VulkanSDK\\1.1.73.0"
-  VULKAN_SDK_URL: https://sdk.lunarg.com/sdk/download/1.1.73.0/windows/VulkanSDK-1.1.73.0-Installer.exe
+  VULKAN_SDK_URL: https://sdk.lunarg.com/sdk/download/1.1.73.0/windows/VulkanSDK-1.1.73.0-Installer.exe?Human=true
```
Comment (AniLeo) [inline appveyor.yml]: Why? We cache this, exceeding download limits is not being an issue anymore.

**Comment by Psycho-A** [comment]
What about OpenGL-only systems and cards? Will the emulator load normally if Vulkan not found at system?

**Comment by kd-11** [comment]
First of all, I do not think psycho-A's issue was really because of vkstatic linking, unless driver loading protocol has changed again. The one benefit of linking against the dll instead of the library is that we do not need to update the precompiled library. However, this will (and I guarantee this will happen) make rpcs3 fail on systems with broken vulkan setup since vulkan-1.dll will likely not be configured correctly. Ironically the solution to this is shipping vulkan-1.dll which contains the same code as vkstatic and has the exact same pitfalls. We switched to vkstatic from vulkan-1 which is the default for this reason.
That said, I'll leave the decision up to those who deal with our #help channel as they will be responsible for diagnosing vulkan library problems with this changeset.

**Comment by Psycho-A** [comment]
Didn't saved from #4625 after I updated video drivers :(

**Pull request merged**